RecoLocalCalo/HGCalRecProducers/plugins/MergeClusterProducer.cc

```diff
@@ -16,22 +16,22 @@ class MergeClusterProducer : public edm::stream::EDProducer<> {
 public:
   /**
    * @brief Constructor with parameter settings - which can be changed in  ...todo.
-   * Constructor will set all variables by input param ps. 
-   * 
+   * Constructor will set all variables by input param ps.
```

Comment:
Is nothing changing in this file except the indentation?
Would you mind rolling back to avoid a spurious diff?

Reply:
@mandrenguyen there are easy ways to hide differences caused only by white-spaces. Given that, I do prefer removing trailing spaces.